src/Ztm.Data.Entity/Contexts/MainDatabase.cs

```diff
@@ -122,7 +122,7 @@ protected virtual void ConfigureWebApiCallback(ModelBuilder modelBuilder)
         {
             modelBuilder.Entity<WebApiCallback>(b =>
             {
-                b.Property(e => e.Id).IsRequired().ValueGeneratedNever();
+                b.Property(e => e.Id).IsRequired().ValueGeneratedOnAdd();
```

Comment:
We should not auto-generate identifier with GUID type due to only some providers don't support it natively.